services/eth_gateway.go

```diff
@@ -715,21 +759,23 @@ func sendETH(from common.Address, fromPrivKey *ecdsa.PrivateKey, toAddr common.A
 	defer cancel()
 
 	// generate nonce
-	nonce, _ := client.PendingNonceAt(ctx, from)
+	nonce, _ := client.PendingNonceAt(ctx, MainWalletAddress)
 
 	// default gasLimit on oysterby 4294967295
 	gasPrice, _ := getGasPrice()
+	// if QAing, uncomment this out and comment out the line above for faster transactions
+	//gasPrice = oyster_utils.ConvertGweiToWei(big.NewInt(5))
 
 	// estimation
-	estimate, failedEstimate := getEstimatedGasPrice(toAddr, from, GasLimitETHSend, *gasPrice, *amount)
+	estimate, failedEstimate := getEstimatedGasPrice(toAddr, MainWalletAddress, GasLimitETHSend, *gasPrice, *amount)
 	if failedEstimate != nil {
 		fmt.Printf("failed to get estimated network price : %v\n", failedEstimate)
 		return types.Transactions{}, "", -1, failedEstimate
 	}
 	estimatedGas := new(big.Int).SetUint64(estimate)
 	fmt.Printf("estimatedGas : %v\n", estimatedGas)
 
-	balance := checkETHBalance(from)
+	balance := checkETHBalance(MainWalletAddress)
```

Comment:
Everywhere in this method where it says "MainWalletAddress" should instead be "fromAddress".  And everywhere where it say "MainWalletPrivateKey" should be "fromPrivKey".  Since we sometimes need to reclaim leftover ETH from other addresses (for which we have the private keys), the caller is now in charge of passing in the from address and private key.